lib/JMAP/Tester.pm

```diff
@@ -334,36 +315,25 @@ sub request {
     $json,
   );
 
-  # Or our sub below leaks us
-  weaken $self;
-
-  $self->ua->set_my_handler(request_send => sub {
-    my ($req) = @_;
-    $self->_logger->log_jmap_request({
-      jmap_array   => \@suffixed,
-      json         => $json,
-      http_request => $req,
-    });
-    return;
+  my $res_f = $self->ua->request($self, $post, jmap => {
```

Comment:
I don't love the logging stuff, it's not clear the `jmap => {}` is for logging. Maybe just a matter of naming. `log_jmap` or something...

Reply:
The logging stuff needs rethinking generally, for sure.  I think it's the most unclear WIP in this WIP.